embedchain/vectordb/chroma.py

```diff
@@ -209,7 +210,7 @@ def reset(self):
         except ValueError:
             raise ValueError(
                 "For safety reasons, resetting is disabled."
```

Comment:
please include a space after the period.

Reply:
Where I didn't get it?